ui/public/js/check-for-IE.js

```diff
@@ -0,0 +1,78 @@
+window.onload = function() {
+    const isIE = !!document.documentMode;
+    if (isIE) {
+      // Adds styling
+      var style = document.createElement('style');
+      style.innerHTML =
+        '#app {' +
+        'position: fixed;' +
+        'top: 50%;' +
+        'left: 50%;' +
+        'width: 35em;' +
+        'height: 20em;' +
+        'margin-top: -11em;' +
+        'margin-left: -18em;' +
+        'text-align: center;' +
+        'border: 1px solid #ccc;' +
+        'background-color: #FFFFFF;' +
+        '}' +
+        'body {' +
+        'background: #f3f3f3;' +
+        '}' +
+        'h2 {' +
+        'margin-top: 1em !important;' +
+        '}' +
+        'img {' +
+        'width: 90px;' +
+        '}' +
+        'a {' +
+        'padding: 0em 1em 0em 1em;' +
+        'text-decoration: none' +
+        '}';
+      document.head.appendChild(style);
+
+      let app = document.getElementById('app');
+      let h1 = document.createElement('h2');
+      let p = document.createElement('p');
+      let imagesDiv = document.createElement('div');
+      imagesDiv.style.cssText += 'display: inline-flex; padding: 0.5em';
+      h1.innerHTML = 'Improve Your Experience';
+      p.innerHTML =
+        "You're using a web browser we don't support.<br>" +
+        'Try one of these options to have a better experience on CERN Library.';
+
+      let imgChrome = document.createElement('img');
+      let aChrome = document.createElement('a');
+      let pChrome = document.createElement('p');
+      imgChrome.src = '/images/chrome-logo.png';
+      aChrome.href = 'https://www.google.com/chrome/';
+      pChrome.innerHTML = 'Google Chrome';
+      aChrome.appendChild(imgChrome);
+      aChrome.appendChild(pChrome);
+
+      let imgFirefox = document.createElement('img');
+      let aFirefox = document.createElement('a');
+      let pFirefox = document.createElement('p');
+      imgFirefox.src = '/images/firefox-logo.png';
```

Comment:
I would probably link to their logo instead of adding is as a static for our website.